sdk/ml/azure-ai-ml/azure/ai/ml/_ml_client.py

```diff
@@ -707,6 +707,7 @@ def from_config(
         """
 
         path = Path(".") if path is None else Path(path)
+        found_path: Any
```

Comment:
`Any` or `Optional[Path]`?

Reply:
I set it to `Optional[Union[Path, str]]` since in line 737, function `traverse_up_path_and_find_file` is retuning `str ` type.